src/webapp/components/ClusterMap.js

```diff
@@ -27,6 +28,14 @@ const makeBounds = (features) => {
   return bounds;
 };
 
+const addDaysOpen = (feature) => ({
+  ...feature,
+  properties: {
+    ...feature.properties,
+    daysOpen: daysSinceSlackMessage(feature.properties.meta.timestamp),
```

Comment:
This works, but I have an idea for another approach: Instead of adding a `daysOpen` property here, we could add a `markerColor` property. This could be computed using a helper that is shared between the map marker and the color chip in the table. This would have two advantages over the existing approach: 1) If we ever change the color coding logic, we only change it in one place, and 2) We get to do the logic in javascript instead of the rather weird Mapbox expression syntax.

Just a thought. If this seems worth doing, let me know :)

Reply:
yes, i think this sounds good, especially for reason 2)!